<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reson_meas_Dlg</class>
 <widget class="QDialog" name="Reson_meas_Dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reasonator Measurement Dialog</string>
  </property>
  <widget class="MplCanvasWrapper" name="mplCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>891</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>410</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Power setup:</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Freq_start_2">
         <property name="text">
          <string>Power (dBm):</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLineEdit" name="edt_Power">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>250</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Frequency setup:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="label_Freq_start">
         <property name="text">
          <string>Start (GHz):</string>
         </property>
         <property name="buddy">
          <cstring>edt_Freq_start</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Freq_stop">
         <property name="text">
          <string>Stop (GHz):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Freq_stop_2">
         <property name="text">
          <string>Center (GHz):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Freq_stop_3">
         <property name="text">
          <string>Span (MHz):</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QLineEdit" name="edt_Freq_start">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_Freq_stop">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_Freq_center">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_Freq_span">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Sweep setup:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="cbBox_swpType">
       <item>
        <property name="text">
         <string>LINear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LOGarithmic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POWer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SEGMent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PHASe</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_swpMode">
       <property name="text">
        <string>Sweep Mode:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbBox_swpMode">
       <item>
        <property name="text">
         <string>SINGle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HOLD </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CONTinuous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GROups</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBox_Avg">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="edt_IF">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_SwpType">
       <property name="text">
        <string>Sweep Type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_AvgTime">
       <property name="text">
        <string>Average times:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_IF">
       <property name="text">
        <string>IF bandwidth (Hz):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_swpPoints">
       <property name="text">
        <string>Sweep Points:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edt_swpPoints">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>490</y>
     <width>281</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Segments (center,span,IF,points,avg):</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="edt_Segment"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_addSeg">
         <property name="text">
          <string>Add Seg.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QListWidget" name="lst_Segments"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delSeg">
         <property name="text">
          <string>Del Seg.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>810</y>
     <width>521</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Save to file:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_dataPath">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edt_dataPath">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_dataFile">
         <property name="text">
          <string>Data File:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="edt_dataFile">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_saveData">
       <property name="text">
        <string>Save Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_figPath">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="edt_figPath">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_figFile">
         <property name="text">
          <string>Fig. File:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="edt_figFile">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_saveFig">
       <property name="text">
        <string>Save Figure</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Resonator Mode Detection:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_searchDept">
          <property name="text">
           <string>Search Depth:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edt_searchDept">
          <property name="text">
           <string>20</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Threshold">
          <property name="text">
           <string>Threshold:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edt_Threshold">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QListWidget" name="lst_resonModes"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QPushButton" name="btn_peakDetect">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Auto Detect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delMode">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Del Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fit results:</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QListWidget" name="lst_Qfactor"/>
      </item>
      <item>
       <widget class="QLabel" name="label_fitFile">
        <property name="text">
         <string>File Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edt_fitFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Export">
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>690</y>
     <width>281</width>
     <height>180</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="btn_singleSweep">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Single Sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_autoSweep">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto Sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Abort">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Abort Sweep</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Quit">
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quit Measurement</string>
    </property>
   </widget>
  </widget>
  <zorder>splitter</zorder>
  <zorder>splitter_2</zorder>
  <zorder>mplCanvas</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>progressBar</zorder>
  <zorder>groupBox_7</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvasWrapper</class>
   <extends>QWidget</extends>
   <header>mplcanvaswrapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edt_Freq_start</tabstop>
  <tabstop>edt_Freq_stop</tabstop>
  <tabstop>edt_Freq_center</tabstop>
  <tabstop>edt_Freq_span</tabstop>
  <tabstop>edt_Power</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Quit</sender>
   <signal>clicked()</signal>
   <receiver>Reson_meas_Dlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>927</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>984</x>
     <y>734</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
